openedx_events/event_bus/avro/tests/test_schema.py

```diff
@@ -243,3 +244,26 @@ def test_throw_exception_to_list_or_dict_types(self):
 
         with self.assertRaises(Exception):
             schema_from_signal(DICT_SIGNAL)
+
+    def test_throw_exception_to_list_or_dict_types_without_annotation(self):
```

Comment:
Isn't this just a duplicate of the above test? Did you mean to rename?

Reply:
@rgraber Thanks for catching it, I indeed meant to rename the function.